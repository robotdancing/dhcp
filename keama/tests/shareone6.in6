# DHCPv6 one-subnet shared-network declaration config

# authoritative is mandatory
authoritative;

# shared-network declaration
shared-network "foobar" {
    # interface
    interface "foo";
    # option
    option dhcp6.domain-search "example.com", "example.org";
    # parameter
    default-lease-time 1800;
    # subnet declaration
    subnet6 2001::/64 {
        # redefined parameter
        default-lease-time 3600;
        # pool
        range6 2001::1000 2001::2000;
    }
    # tried another pool here but DHCPv6 pools are allowed only in subnets
}
