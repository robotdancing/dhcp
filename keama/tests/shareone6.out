{
  # DHCPv6 one-subnet shared-network declaration config
  # authoritative is mandatory
  "Dhcp6": {
    "interfaces-config": {
      "interfaces": [
        "en0"
      ]
    },
    "subnet6": [
      # subnet declaration
      {
        "subnet": "2001::/64",
        "valid-lifetime": 3600,
        "pools": [
          {
            # pool
            "pool": "2001::1000 - 2001::2000"
          }
        ],
        "interface": "en0",
        "option-data": [
          # interface
          # option
          {
            "space": "dhcp6",
            "name": "domain-search",
            "code": 24,
//          "original-data": "\"example.com\", \"example.org\"",
            "data": "example.com, example.org"
          }
        ]
      }
    ]
  }
}
